content/en/docs/reference/access-authn-authz/abac.md

```diff
@@ -16,12 +16,12 @@ Attribute-based access control (ABAC) defines an access control paradigm whereby
 {{% capture body %}}
 ## Policy File Format
 
-For mode `ABAC`, also specify `--authorization-policy-file=SOME_FILENAME`.
+To enable `ABAC` mode, specify `--authorization-policy-file=SOME_FILENAME` on startup.
```

Comment:
Specifying that file does not enable ABAC, it is required in addition to --authorization-mode=ABAC. The previous text was more accurate